.github/workflows/publish-packages.yml

```diff
@@ -0,0 +1,20 @@
+name: Publish to crates.io
+
+on:
+  push:
+    branches:
+      - main
+
+
```

Comment:
usually, we have just one line between sections

Reply:
fixed